src/main/java/com/snack/news/dto/RequestQueryDto.java

```diff
@@ -0,0 +1,36 @@
+package com.snack.news.dto;
+
+import lombok.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'lombok'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)